api/src/shop/stripe_constants.py

```diff
@@ -1,44 +1,90 @@
+from enum import Enum
 import stripe
+from test_aid.systest_config import STRIPE_PRIVATE_KEY, STRIPE_PUBLIC_KEY
 
 from service.config import config
 
-stripe.api_key = config.get("STRIPE_PRIVATE_KEY", log_value=False)
-
 # All stripe calculations are done with cents (ören in Sweden)
 STRIPE_CURRENTY_BASE = 100
 
 CURRENCY = "sek"
 
 STRIPE_SIGNING_SECRET = config.get("STRIPE_SIGNING_SECRET", log_value=False)
 
+def set_stripe_key(private: bool) -> None:
+    stripe.api_key = STRIPE_PRIVATE_KEY if private else STRIPE_PUBLIC_KEY
+
+set_stripe_key(True)
+
+class MakerspaceMetadataKeys(Enum):
+    USER_ID = "makerspace_user_id"
+    MEMBER_NUMBER = "makerspace_member_number"
+    PRICE_TYPE = "price_type"
+    SUBSCRIPTION_TYPE = "subscription_type"
+    TRANSACTION_IDS = "makerspace_transaction_ids"
+    PENDING_MEMBER = "makerspace_pending_member"
+
+    # Used in the webshop for special products
+    SPECIAL_PRODUCT_ID = "special_product_id"
```

Comment:
Can't these just be separate columns in the webshop item instead? Having unstructured data in a database column sounds more painful to migrate in the future. And requires custom validation on modifications?